### DISCUSSION TITLE
Ongoing Tasks

### DISCUSSION THREAD

**Discussion by shivaylamba** [mexili/incogly#119] [General]
- Peer.js Server Setup ( @raghavdhingra ) 
- Audio Modulation ( @ZohebMOPO )
- Landing Page Chakra UI Refactor ( @shivaylamba ) 
- ML integration ( @shivaylamba @midopooler )
- Architecture ( @raghavdhingra @sansyrox )

**Comment by sansyrox** [comment]
@shivaylamba , landing page is already implemented in chakra ui. 
We need to implement the video call screen with chakra
